src/components/dialogmote/innkalling/BehandlerRadioGruppe.tsx

```diff
@@ -7,7 +7,7 @@ import { capitalizeWord } from "@/utils/stringUtils";
 const texts = {
   behandlerLegend: "Behandler som inviteres til dialogmøtet",
   behandlerInfo:
-    "Fastlegen vil få en dialogmelding med invitasjon. Dersom du ønsker å invitere en annen behandler, må du fortsatt sende innkallingen fra Arena.",
+    "Behandleren vil få en dialogmelding med invitasjon. Dersom du ønsker å invitere en annen behandler, må du fortsatt sende innkallingen fra Arena.",
```

Comment:
Vi trenger kanskje ikke den andre setningen nå? @torhrise

Reply:
Enig, den kan utgå.

Reply:
Entall her referer jo til den ene, valgte behandleren (enten i modia, eller i Arena), så man trenger vel ikke skrive noe annet selv om det finnes flere i listen? 🤔

Reply:
Mulig jeg henger meg litt opp i teksten her, men den sier i praksis at dersom man vil kalle in en behandler som ikke er valgt , så skal man kalle inn i Arena. Dvs. at dersom det er 3 i listen, så ber teksten deg om å kalle inn i Arena for 2 av 3.

Reply:
Ja, da skjønner jeg hva du mener! 👍 Det kommer jo an på hvordan man tolker `...invitere en annen behandler`, det kan jo referere både til `annen behandler [enn den som er valgt i forrige setning]` og `annen behandler [enn de som finnes i denne konteksten]` 🤔 Men det ble bra med den endringen June gjorde nå!

Reply:
Jepp, enig 👍